pkg/controller/configmap/configmap_controller.go

```diff
@@ -496,6 +500,25 @@ func (r *ReconcileConfigMap) updateSharedInfoWithNsxNcpResources(objs []*unstruc
 	log.Info("Updated shared info with Nsx Ncp Resources")
 }
 
+func (r *ReconcileConfigMap) isAnyPodMonitoredNCPResDeleted() (bool, error) {
```

Comment:
A very pedant comment: you can probably remove "Pod" from this func's name, since the function does not monitor pods after all?

Reply:
right, updated